src/customMetadata/GS_Admin_Checklist_Item.NPSP_Settings.md

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>NPSP Settings</label>
+    <protected>true</protected>
+    <values>
+        <field>Description_Label__c</field>
+        <value xsi:type="xsd:string">gsChecklistItemNPSPSettingsDesc</value>
+    </values>
+    <values>
+        <field>Extra_Info_Label__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>GS_Admin_Checklist_Section__c</field>
+        <value xsi:type="xsd:string">Make_it_your_own</value>
+    </values>
+    <values>
+        <field>Has_Link__c</field>
+        <value xsi:type="xsd:boolean">false</value>
+    </values>
+    <values>
+        <field>Image__c</field>
+        <value xsi:type="xsd:string">settings_illustration.svg</value>
+    </values>
+    <values>
+        <field>Link_Label__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>Link_URL__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>Position__c</field>
+        <value xsi:type="xsd:double">4.0</value>
+    </values>
+    <values>
+        <field>Primary_Button_Label__c</field>
+        <value xsi:type="xsd:string">gsChecklistItemNPSPSettingsPriBtnLabel</value>
```

Comment:
@smarovich The button label for this is "Adjust Settings", but the button action takes the user to a PDF. It just seems odd.
![image](https://user-images.githubusercontent.com/801017/100263448-85c78500-2f1b-11eb-9ea0-d80cf18c1493.png)

Reply:
@force2b the WI related to that subsection is: W-7976392

Reply:
Thanks @yiselhernandezSF. Let's let @smarovich weight in on Monday.

Reply:
Per [W-7976392](https://gus.lightning.force.com/lightning/r/0D5B000001GwmeIKAR/view), we decided to go with "Learn More" for the button.

Reply:
@force2b This was changed in PR: https://github.com/SalesforceFoundation/NPSP/pull/6081